containerd/socket-tracer/containerd-socket-tracer.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-socket-tracer
+  namespace: default
+  labels:
+    k8s-app: containerd-socket-tracer
+spec:
+  selector:
+    matchLabels:
+      app: containerd-socket-tracer
+  template:
+    metadata:
+      labels:
+        app: containerd-socket-tracer
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+      - name: tracer
+        image: ubuntu:25.04
```

Comment:
use GKE-hosted image

Reply:
What image would you suggest? The closest thing I could find on gke.gcr.io was `debian-base` however `bpftrace` did not work with this image. We're actually using `ubuntu` (no tag) elsewhere in this repo. I'd prefer to go with that unless we have a suitable replacement.

Reply:
ok, we have a mirror.gcr.io configured. So should be reasonable.

We can always do mirror.gcr.io/ubuntu which will make the usage of a mirror explicit =)